scripts/build_tools.sh

```diff
@@ -17,6 +17,6 @@ export CGO_ENABLED=0
 # Build binary
 cd $GOPATH/src/github.com/raintank/metrictank/cmd
 for tool in *; do
-  cd $tool
+  cd $GOPATH/src/github.com/raintank/metrictank/cmd/$tool
```

Comment:
do we still want this change? b89b20e4f9ecb4195c7240f590ad3a8772f6ba95 should fix the same problem. we can probably just remove this commit